File: testing/highlight-samples/sample.zig
// This is a comment
const std = @import("std");

const User = struct {
    name: []const u8,
    age: i32 = 42,
};

fn greet(user: User) void {
    std.debug.print("Hi, {s}!\n", .{user.name});
}

const pi: f64 = 3.14;
const ch: u8 = 'A';

File: testing/highlight-samples/sample.rs
// This is a comment
use std::fmt;

struct User {
    name: String,
    age: i32,
}

fn greet(user: &User) -> String {
    format!("Hi, {}!", user.name)
}

let count: i32 = 42;
let pi: f64 = 3.14;
let active: bool = true;
let ch: char = 'a';

/* Block comment */
